### DISCUSSION TITLE
Is it possible to support images in jfif format?

### DISCUSSION THREAD

**Discussion by ghost** [tweepy/tweepy#1699] [Questions]
Hello.

This is my first time asking a question, so I apologize if this is weird.
I hired another engineer to use tweepy, and he had a problem where he couldn't download some images.
The reason seems to be that it doesn't support jfif, is it possible to fix it?

Sorry. I'm not an engineer, so I don't know the details.
It would be very helpful if you could help.

**Comment by gregyjames** [comment]
What exactly is the issue that he has with downloading the images? If they are showing in the entities object of the status, he could try downloading those images from those URL directly. If the issue is with the images not showing up in the object he could use Tweepy to get the id's for the specified tweets then use a get request to twitters API and return the status manually.

**Comment by Harmon758** [comment]
I'm not sure what support for JFIF images means here.
I don't think the Twitter API uses JFIF images anywhere, Tweepy itself does not provide a specific means of downloading images (you can do so using the URLs provided by the Twitter API / Tweepy), and generally, there doesn't need to be "support" for specific file formats when downloading them.

**Comment by ghost** [comment]
Thanks for the advice.
Sorry, my sentence is bad.  I got the source code that is the error, so I will post it.


▽ "tweepy.Cursor" method can't get the result of jfif image url.  What's wrong? 

```
 search_results = tweepy.Cursor(api.user_timeline, screen_name=key_account, include_rts=False).items(count_no)

for result in search_results:
    if hasattr(result, 'extended_entities'): #resultが'extended_entities
        ex_media = result.extended_entities['media']
        tweet_id = result.id
        if 'video_info' in ex_media[0]:
            ...
        else:
            #image
            n = 1
            for image in ex_media:
                image_url = image['media_url']
                n = n + 1
```

**Reply by Harmon758** [reply]
Is there an error?
